### DISCUSSION TITLE
Version Bug in "About" Setting

### DISCUSSION THREAD

**Discussion by jmindev22** [open-webui/open-webui#1697] [General]
I have a bug in about's section since v0.1.117, this bug consists in not changing the version (text) even updating open webui and showing the message that is the latest version, does anyone have the same problem? Is there any way to solve it? This bug is only visual since I can use the newest update funcionalities
![image](https://github.com/open-webui/open-webui/assets/164365149/5deaf08e-eeab-4e62-9c53-c44d41eae676)

**Comment by justinh-rahb** [comment]
You've got a stubborn cache somewhere in your network stack between your browser and your server most likely that just won't let go. Also, you're ***certain*** you've updated? The server prints it's version in the console/docker log, as does the frontend in the browser console.

```bash
docker logs open-webui
```

**Reply by tjbck** [reply]
Strange bug indeed 😅

**Reply by jmindev22** [reply]
Definitely I have updated it correctly since the newest version it displays when I run the bash code (for example), it is strange that it only shows in some places such as this one.

**Comment by JuanjoJG-WATA** [comment]
I'm having the same problem. I've updated it, and it still shows v0.1.117, even tho every time I log in with any of my admin test accounts, the pop-up shows that the v0.1.120 is the one I'm using ATM.
![image](https://github.com/open-webui/open-webui/assets/165764759/53552d43-8271-4988-8fe9-9c28c3bffd2d)

I have no idea why it happened, but it's been like that since I updated to the v0.1.118.

**Reply by tjbck** [reply]
Method of installation?

**Reply by JuanjoJG-WATA** [reply]
I followed every step in: https://github.com/open-webui/open-webui/tree/functions?tab=readme-ov-file#how-to-install-without-docker

That's the only weird thing I've found so far.

**Reply by justinh-rahb** [reply]
We may at least be able to rule out caching issues... is there another URL you could try reaching it at? Like if you had been using localhost:3000 try 127.0.0.1:3000 or vice-versa, or the LAN IP of the host.

Failing that, my next course would be to remove the `node_modules` and any Python virtual environment you may have (recommended for direct installs) and attempting to reinstall from a cleaner slate.

I might go so far as to re-clone the whole repo (take care that you don't lose your `/backend/data` contents).
